libs/product/src/models/product.ts

```diff
@@ -1,11 +1,17 @@
 import { DaffProductImage } from './product-image';
 
+export enum DaffProductTypeEnum {
+	SimpleProduct = 'Simple Product',
+	BundledProduct = 'Bundled Product',
```

Comment:
Let's instead call this a `composite` product. The types here should be shorter:

```ts
Simple = 'simple',
Variant = 'variant',
Composite = 'composite'
```

Reply:
We actually don't support `variant` products yet, so I'd drop that type as well.